src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,393 @@
+﻿/* @internal */
+namespace ts.codefix {
+
+    registerCodeFix({
+        errorCodes: [Diagnostics.Cannot_find_name_0.code],
+        getCodeActions: (context: CodeFixContext) => {
+            const sourceFile = context.sourceFile;
+            const checker = context.program.getTypeChecker();
+            const allSourceFiles = context.program.getSourceFiles();
+            const useCaseSensitiveFileNames = context.host.useCaseSensitiveFileNames ? context.host.useCaseSensitiveFileNames() : false;
+
+            const token = getTokenAtPosition(sourceFile, context.span.start);
+            const name = token.getText();
+            const allActions: CodeAction[] = [];
+
+            const allPotentialModules = checker.getAmbientModules();
+            for (const otherSourceFile of allSourceFiles) {
+                if (otherSourceFile !== sourceFile && otherSourceFile.symbol) {
+                    allPotentialModules.push(otherSourceFile.symbol);
+                }
+            }
+
+            const currentTokenMeaning = getMeaningFromLocation(token);
+            for (const moduleSymbol of allPotentialModules) {
+                context.cancellationToken.throwIfCancellationRequested();
+
+                const moduleExports = checker.getExportsOfModule(moduleSymbol);
+                if (!moduleExports) {
+                    continue;
+                }
+
+                // check the default export
+                const defaultExport = moduleExports["default"];
+                if (defaultExport) {
+                    const localSymbol = getLocalSymbolForExportDefault(defaultExport);
+                    if (localSymbol && localSymbol.name === name && checkSymbolHasMeaning(localSymbol, currentTokenMeaning)) {
+                        allActions.push(getCodeActionForImport(moduleSymbol, /*isDefaultExport*/ true));
+                    }

```

Comment:
I don't have data, but a quick search in the DefinitelyTyped repo did give me some library files with named `export default`, for example the [`material-ui` library](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/23d8466f82f9397d4d5e7cfe709439def66e8a39/material-ui/index.d.ts#L1913). And in the latest implementation I return multiple code fixes to the user, so that he/she can choose which one he/she likes the best.